.github/workflows/component-builder.yml

```
@@ -1,19 +1,29 @@
 name: "Component Builder Check"
 on:
-  pull_request:
+  # Wait for event on pull_request_target so that this action
+  # can access repo secrets: QUAY_USER & QUAY_TOKEN
+  pull_request_target:
+    permissions:
```

actionlint (1.7.7)

unexpected key "permissions" for "pull_request_target" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)